backend/src/bin/scripts/import-lfx-memberships.ts

```diff
@@ -101,6 +101,11 @@ if (parameters.help || !parameters.file || !parameters.tenantId) {
     const qx = SequelizeRepository.getQueryExecutor({
       database: prodDb,
     } as IRepositoryOptions)
+
+    await qx.result(`TRUNCATE "lfxMemberships"`)
```

Comment:
This can be dangerous if there are other tables *depending* on this one, and the will be truncated too.
Let's change it to `DELETE FROM "lfxMemberships"`, so if we actually have some dependencies, we know about it, because this query starts to fail.